eth/main.cpp

```diff
@@ -605,6 +606,15 @@ int main(int argc, char** argv)
 			chainParams = ChainParams(genesisInfo(eth::Network::Olympic));
 		else if (arg == "--morden" || arg == "--testnet")
 			chainParams = ChainParams(genesisInfo(eth::Network::Morden), genesisStateRoot(eth::Network::Morden));
+		else if (arg == "--oppose-dao-fork")
+		{
+			chainParams = ChainParams(genesisInfo(eth::Network::Frontier), genesisStateRoot(eth::Network::Frontier));
+			chainParams.otherParams["daoHardforkBlock"] = toHex(u256(-1) - 10, HexPrefix::Add);
+		}
+		else if (arg == "--support-dao-fork")
```

Comment:
Let's just omit this, eh?    If we are defaulting to hard-fork then there is no need for this setting.   It will just hang around and do nothing, and then get stripped out again at some point in the future.

Better just to not add it in the first place.